src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/SblInstanceEvent.cs

```diff
@@ -0,0 +1,82 @@
+using Newtonsoft.Json;
```

Comment:
I don't see that the nessesary changes are made in the controllermethod called by SBL. 
Currently, SBL uses instanceEventsController to get the list of events. 
With this change an endpoint should probably exist in messsageBoxInstancesController that returns List<SBLInstanceEvent>

Reply:
moved to platform.storage helpers